operators/pkg/bastion-controller/bastion_controller.go

```diff
@@ -50,7 +50,7 @@ func (r *BastionReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 	klog.Info("reconciling bastion")
```

Comment:
Unrelated from this PR. Could you add the name of the tenant that is being reconciled? Otherwise this log is useless.

Reply:
Should I put it in this PR then?

Reply:
Yep, would avoid opening a PR just for one log line